chore(windows): cleanup resources 🎢

mcdurdin opened the pull request:
Moves shared resources to common, Developer resources to developer, and Windows-only resources to windows/src/desktop/branding. Removes old, unused resources, and cleans up references.

This will be most easily reviewed commit-by-commit, because the changes are grouped more neatly this way.

@keymanapp-test-bot skip

ermshiperete commented:
`common/resources/icons/filetypes/Thumbs.db` - is this actually used/needed by our code or is that just the Windows Explorer thumbnail database?

mcdurdin commented on windows/src/desktop/inst/keymandesktop.wxs:
```diff
@@ -126,11 +126,11 @@
 
               <Component>
-                <File Id="appicon.ico" Name="appicon.ico" Source="..\..\global\res\90\desktop\appicon.ico" KeyPath="yes" />
+                <File Id="appicon.ico" Name="appicon.ico" Source="..\..\desktop\branding\appicon.ico" KeyPath="yes" />
```
Cleaned up, thanks!

mcdurdin commented:
> `common/resources/icons/filetypes/Thumbs.db` - is this actually used/needed by our code or is that just the Windows Explorer thumbnail database?

Yeah, it's a Windows Explorer thing... it should be git-ignored globally. Removed and added entry to .gitignore./

jahorton commented:
New commits look good and clean.
